packages/kbn-search-api-panels/components/install_client.tsx

```diff
@@ -73,21 +73,6 @@ export const InstallClientPanel: React.FC<InstallClientProps> = ({
       />
       <EuiSpacer />
       <Link language={language} assetBasePath={assetBasePath} />
-      <EuiSpacer />
```

Comment:
question: Did we want this removed entirely?

Reply:
We had two callouts related to "try in console" so julian asked for it to be removed.